src/screens/VaultScreen.js

```diff
@@ -4,8 +4,16 @@ import { Font } from 'expo';
 import Icon from '@expo/vector-icons/FontAwesome';
 import { imageStyles } from '../styles/images';
 import { layoutStyles } from '../styles/layout';
+import ResourceRequester from '../helpers/requesters/ResourceRequester';
+import UpvoteRequester from '../helpers/requesters/UpvoteRequester';
 
 export default class VaultScreen extends React.Component {
+  static navigationOptions = {
+    tabBarLabel: 'Vault',
+    tabBarIcon: ({ tintColor }) => (
+      <Icon name="briefcase" size={22} color="#e91e63" />
+    ),
+  };
 
```

Comment:
You should specify the Navigation options in the navigator and change the color={tintColor}